images/openwisp_base/Dockerfile

```diff
@@ -61,10 +63,11 @@ ARG OPENWISP_USERS_SOURCE=default
 RUN if [ "$OPENWISP_USERS_SOURCE" != "default" ] ; then \
         pip install --user --upgrade --force-reinstall ${OPENWISP_USERS_SOURCE}; \
     fi
-ARG OPENWISP_UTILS_SOURCE=default
+# TODO: Remove when next version of openwisp-utils is released
+ARG OPENWISP_UTILS_SOURCE="openwisp-utils[celery,rest] @ https://github.com/openwisp/openwisp-utils/tarball/master"
```

Comment:
let's use specific commits for these deps to avoid suprises and frequent failing builds in the coming iterations we'll do here